### PR TITLE
[FEAT] Added September blog 

### PR DESCRIPTION
**Pull request by Saurabha999**

### CODE REVIEW

File: post/blogs/2023/september_2023/index.md
Comment (kunalagra) [inline post/blogs/2023/september_2023/index.md:1]: Change the author tuple to reflect the new Authors, Lastmod cannot be lower than date, add relevant description for images for higher accessibility.

**Comment by Saurabha999** [comment]
- [x] Change the author tuple
- [x] Changed position of date and Lastmod
- [x] Better naming of images for higher accessibility

**Comment by kunalagra** [comment]
lastmod is the metadata that signifies the date when the article was last modified.
So the date should match the date or should be greater than its value.

**Comment by Saurabha999** [comment]
Lastmod and Date have same date

**Pull request merged**